src/geometry/GeometryEvaluator.cc

```diff
@@ -1130,12 +1130,12 @@ static std::unique_ptr<Geometry> extrudePolygon(const LinearExtrudeNode& node, c
 
   const Polygon2d& polyref = is_segmented ? seg_poly : poly;
 
-  double h1, h2;
+  Vector3d h1, h2;
   if (node.center) {
-    h1 = -node.height / 2.0;
-    h2 = +node.height / 2.0;
+    h1 = Vector3d(-node.height[0] * 0.5, -node.height[1] * 0.5, -node.height[2] * 0.5);
+    h2 = Vector3d(node.height[0] * 0.5, node.height[1] * 0.5, node.height[2] * 0.5);
```

Comment:
You shouldn't need to change this. Eigen Vector3d supports the `/` operator.

Reply:
Yep, only unary plus operator is not supported by Eigen